src/com/sun/ts/tests/jpa/core/types/field/Client.java

```diff
@@ -1018,6 +1019,40 @@ public void fieldTypeTest17() throws Fault {
       throw new Fault("fieldTypeTest17 failed");
   }
 
+  /*
+   * @testName: testCreateUUIDType
+   *
+   * @assertion_ids: PERSISTENCE:SPEC:527;
+   *
+   * @test_Strategy: Test EM create on entity class with UUID.
+   */
+  public void testCreateUUIDType() throws Fault {
+    UUID id = UUID.randomUUID();
+    UUIDType uuidType = new UUIDType(id, "Create UUID Type");
+    try {
+      getEntityTransaction().begin();
+      getEntityManager().persist(uuidType);
+      getEntityManager().flush();
```

Comment:
As per[ comment about adding call to `EntityManager.clear()`](https://github.com/eclipse-ee4j/jakartaee-tck/pull/793#issuecomment-1011295379), should a call to `getEntityManager().clear()` be added here?

Reply:
Done.